anoncreds/protocol/types.py

```diff
@@ -193,6 +193,35 @@ def __eq__(self, other):
                and self.Z == other.Z and self.seqId == other.seqId \
                and dict(self.R) == dict(other.R)
 
+    def to_str_dict(self):
```

Comment:
- Please add tests that these two new methods work (for example, similar to testPKFromToDict)
- Please add a test to check that serialization equals to libsovrin's format